packages/expo-app-loading/src/SplashScreen.expo.ts

```diff
@@ -0,0 +1,15 @@
+import { NativeModules } from 'react-native';
+
+const { ExponentSplashScreen: SplashScreen = {} } = NativeModules;
+
+export function preventAutoHideAsync(): void {
+  if (SplashScreen.preventAutoHide) {
+    SplashScreen.preventAutoHide();
+  }
+}
+
+export function hideAsync(): void {
+  if (SplashScreen.hide) {
+    SplashScreen.hide();
+  }
+}
```

Comment:
We have expo-splash-screen -- this looks like duplicate functionality

Reply:
Yes, but it's not (yet) fully working in managed. This connects to the "old" `import { SplashScreen } from 'expo';` API to make it work in managed. I think bbarthec is working hard to make `expo-splash-screen` possible in managed. I'll put a notice in the PR that this is waiting for that, and refactor the code later 😄